templates/bot/js/notification-function-base/README.md

```diff
@@ -1,43 +1,37 @@
-# Send notifications to Teams
+# Overview of the Notification bot template
 
-The Notification in Teams feature enables you to consume, transform  and post events as plain text or [adaptive cards](https://docs.microsoft.com/microsoftteams/platform/task-modules-and-cards/cards/cards-reference) to an individual, chat, or channel in Teams.
+This template showcases an app that send a message to Teams with Adaptive Cards triggered by a HTTP post request. You can further extend the template to consume, transform and post events to individual, chat or channel in Teams.
```

Comment:
It seems the *Adaptive Cards* and *Microsoft Bot Framework* links are removed. Just double confirm is that expected?

Reply:
Yes, was intentional. Keep it more simplified and consistent with other READMEs.